Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Turing"
 uuid = "fce5fe82-541a-59a6-adf8-730c64b5f9a0"
-version = "0.13.0"
+version = "0.13.1"
```

Comment:
We should make sure that we haven't introduced any breaking changes since 0.13.0. (IMO we should adopt the ColPrac practice of making patch releases for every PR).

Reply:
I'll just bump it up to 0.14.0. Honestly at this point we should consider moving to 1.0 as well.

Reply:
More importantly, current 0.13.0 is by default failing so `]  add Turing` and `using Turing` will fail. I think maybe you guys want to bump up the version really soon...

Reply:
We first have to fix the bug introduced by the changes in PDMats 0.10 on master before releasing 0.14.0. What was your package setup that failed, i.e. can you post the output of `] st`?

Reply:
Oh, I have mine on Turing#master, but there are more than one guys on slack that faces an issue: `] add Turing` installs an older version and `using Turing` somehow fails.

Reply:
Usually these problems are caused by unbounded compatibilities of old Turing versions (there are many closed issues in the repo here). These issues should be fixed by running `] add Turing@0.13.0` and possibly adjusting conflicting packages (by users) and adding correct bounds in the registry (by us). I fixed some bounds a while ago, but it seems the old version are still missing some compatibilty bounds.